File: tourneyAPI/Models/Entities/Characters/Link.cs
namespace Entities;

using Enums;


// Defines the competitive profile metadata for this playable character.
public class Link : Character
{
    // Initializes this character's default competitive attributes for matchmaking and tier logic.
    public Link()
    {
        Id = CharacterId.Link;
        characterName = CharacterName.LINK;
        archetype = Archetype.TURTLE;
        fallSpeed = FallSpeed.FAST_FALLERS;
        weightClass = WeightClass.MIDDLEWEIGHT;
        tierPlacement = TierPlacement.C;
    }

}

File: tourneyClient/src/models/entities/Characters/Link.ts
import { CharacterId } from "@/models/Enums/CharacterId";
import { Character } from "../Character"
import { CharacterName } from "@/models/Enums/CharacterName";
import { Archetype } from "@/models/Enums/Archetype";
import { FallSpeed } from "@/models/Enums/FallSpeed";
import { WeightClass } from "@/models/Enums/WeightClass";
import { TierPlacement } from "@/models/Enums/TierPlacement";

export const Link: Character =
{
    id: CharacterId.Link,
    characterName: CharacterName.LINK,
    archetype: Archetype.TURTLE,
    fallSpeed: FallSpeed.FAST_FALLERS,
    weightClass: WeightClass.MIDDLEWEIGHT,
    tierPlacement: TierPlacement.C
}

export default Link; 